monitoring/instrumented_mutex.h

```diff
@@ -49,6 +61,9 @@ class InstrumentedMutex {
   Statistics* stats_;
   SystemClock* clock_;
   int stats_code_;
+#ifdef COERCE_CONTEXT_SWITCH
+  InstrumentedCondVar* bg_cv_;
```

Comment:
You could put `= nullptr` here and then not have to modify all the old constructors with conditionally compiled code.

Reply:
Good point!

Reply:
Fixed